install.js

```diff
@@ -414,7 +422,22 @@ var steps = [
   },
   // all done
   function finalize (next) {
-    console.log("Installation complete.\nRun the command 'node server' (or 'foreman start' if using heroku toolbelt) to start your instance.");
+    var passedArgs = process.argv.slice(2);
+
+    for (var i = 2; i < passedArgs.length; i++) {
+      if (passedArgs[i] == undefined) {
+        isVagrantInstallation = false;
+        break;
+      } else {
+        isVagrantInstallation = true;
+      }
+    }
+
+    if (isVagrantInstallation) {
```

Comment:
You're looping over each argument here and setting `isVagrantInstallation = true` multiple times.  It's probably better to just do something like:

`isVagrantInstallation = process.argv.length == 2 ? false : true;`

Also, can you rename this variable to `isVagrant`, for brevity?  The 'installation' part can be inferred from the fact that you're in the installation script (install.js).

You could also move this check further up the script (at the start) so other pieces of the code can know if if it's a Vagrant install.